src/MICore/LaunchOptions.cs

```diff
@@ -1898,6 +1894,8 @@ public void InitializeLaunchOptions(Json.LaunchOptions.LaunchOptions launch)
             {
                 this.LaunchCompleteCommand = launch.LaunchCompleteCommand.Value;
             }
+
+            this.Environment = new ReadOnlyCollection<EnvironmentEntry>(GetEnvironmentEntries(launch.Environment) ?? new List<EnvironmentEntry>());
```

Comment:
?? will never return false since GetEnvironmentEntries seems to always return a non-null.